lib/autotune/index.js

```diff
@@ -203,28 +208,35 @@ function generate (inputs) {
             mealCarbs = Math.max(mealCarbs, parseInt(csf_glucose[i].mealCarbs));
         }
     }
+    // at midnight, write down the mealcarbs as total meal carbs (to prevent special case of when only one meal and it not finishing absorbing by midnight)
+    // TODO: figure out what to do with dinner carbs that don't finish absorbing by midnight
     if (totalMealCarbs == 0) { totalMealCarbs += mealCarbs; }
     if (totalDeviations == 0) { totalDeviations += deviations; }
-    //console.error(totalDeviations, totalMealCarbs);
+    //console.error(totalDeviations, totalMealCarbs)
```

Comment:
Probably should revert or remove this, so it doesn't syntax error if someone uncomments it.